explore/future_compat.h

```diff
@@ -62,12 +62,16 @@
     #define VW_WARNING_DISABLE_DEPRECATED_USAGE    VW_WARNING_DISABLE(4996)
 #  define VW_WARNING_DISABLE_BADLY_FORMED_XML VW_WARNING_DISABLE(4635)
 #  define VW_WARNING_DISABLE_CPP_17_LANG_EXT VW_WARNING_DISABLE(4984)
+#  define VW_WARNING_DISABLE_COND_CONST_EXPR VW_WARNING_DISABLE(4127)
+#  define VW_WARNING_DISABLE_HIDE_CLASS_MEMBER VW_WARNING_DISABLE(4458)
```

Comment:
I dont think this is a warning we want to suppress

Reply:
Removed.